reference-architecture/rhv-ansible/example/vars/ovirt-37-vars.yaml

```diff
@@ -0,0 +1,157 @@
+---
+###########################
+# Common
+###########################
+compatibility_version: 4.1
+
+# Data center
+data_center_name: Default
+
+##########################
+# VM infra
+##########################
+template_cluster: "{{ rhv_cluster }}"
+template_name: centos7
+template_memory: 8GiB
+template_cpu: 1
+template_disk_storage: "{{ rhv_data_storage }}"
+template_disk_size: 60GiB
+template_nics:
+  - name: nic1
+    profile_name: ovirtmgmt
+    interface: virtio
+
+##########################
+# Other top scope vars
+##########################
+debug_vm_create: true
+
+master_vm:
+  cluster: "{{ rhv_cluster }}"
+  template: "{{ template_name }}"
+  memory: 16GiB
+  cores: 2
+  high_availability: true
+  disks:
+    - size: 100GiB
+      storage_domain: "{{ rhv_data_storage }}"
+      name: docker_disk
+      interface: virtio
+
+node_vm:
+  cluster: "{{ rhv_cluster }}"
+  template: "{{ template_name }}"
+  memory: 8GiB
+  cores: 2
+  disks:
+    - size: 100GiB
+      storage_domain: "{{ rhv_data_storage }}"
+      name: docker_disk
+      interface: virtio
+    - size: 50GiB
+      storage_domain: "{{ rhv_data_storage }}"
+      name: localvol_disk
+      interface: virtio
+
+
+hack_guest_agent_script: |
```

Comment:
any reason behind this?

Reply:
RHV 4.2 sets up a different port, so it has to be modified in existing RHEL 7.4 guest images.

Reply:
I'd say then have some comment explaining it instead just hack_guest_agent_script :D